src/service/viewer-impl.js

```diff
@@ -74,6 +74,12 @@ export const VisibilityState = {
 };
 
 
+/** @export {!Array<string>} */
+export const TRUSTED_VIEWER_HOSTS = [
+  'google.com'
```

Comment:
This is where it gets complicated due to TLDs.

I'm not sure there is a way to do this without a full list of TLDs :(